specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspAccessRulePut.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Create/Update NSP access rule",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1",
+    "profileName": "profile1",
+    "accessRuleName": "accessRule1",
+    "parameters": {
+      "properties": {
+        "direction": "Inbound",
+        "addressPrefixes": [
+          "10.11.0.0/16",
+          "10.10.1.0/24"
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "accessRule1",
+        "id": "/subscriptions/subId/resourceGroup/rg1/providers/Microsoft.Network/networkSecurityPerimeters/nsp1/profiles/profile1/accessRules/accessRule1",
+        "type": "Microsoft.Network/networkSecurityPerimeters/profiles/accessRules",
+        "properties": {
```

Comment:
provisioningState - to have consistent response across CRUD operations

Reply:
Will remove once we remove from the backend models.